app/src/main/java/com/bogdan/codeforceswatcher/activity/MainActivity.kt

```diff
@@ -90,7 +94,7 @@ class MainActivity : AppCompatActivity() {
         private val mFragmentTitleList = arrayListOf("Users", "Contests")
 
         override fun getItem(position: Int): Fragment {
-            return mFragmentList[position]
+            return mFragmentList[position] as Fragment
```

Comment:
Shouldn't it be the list of `Fragment`s? So when you are taking any element from it, it's already a `Fragment`.